extensions/cornerstone/src/commandsModule.ts

```diff
@@ -404,16 +406,6 @@ const commandsModule = ({ servicesManager }) => {
         (activeViewportIndex - 1 + viewports.length) % viewports.length;
       viewportGridService.setActiveViewportIndex(nextViewportIndex);
     },
```

Comment:
not sure why `setHangingProtocol` is deleted, maybe it has a reason that I have not reached yet in the rest of the code changes. But, we need a way to set it explicitly by Id (e.g., next button that goes to next hanging protocol)